Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG ARGOCD_VERSION=latest
 FROM argoproj/argocd:$ARGOCD_VERSION as argocd
 
-FROM node:11.10.1-slim
+FROM node:12.19.0-buster
```

Comment:
What's the reason behind using buster vs slim?

Reply:
I will try this with slim instead. I was using buster since it had some debugging tools on it that I was using.